src/index.js

```diff
@@ -293,3 +293,9 @@ module.exports = {
   }
 };
 
+var OutboundLink = require('./components/OutboundLink');
+OutboundLink.trackLink = reactGA.outboundLink;
```

Comment:
the more I look at this, the more it makes sense to rename the func. in reactGa to `trackLink`, too. That way we don't have "the same word, just with different case" inside the reactGA object